cmd/experimental/podtaintstolerations/README.md

```diff
@@ -0,0 +1,121 @@
+# Pod Taints & Tolerations Controller
+
+This controller adds experimental support for bare Pod workloads, targeting clusters that predate support for [scheduling gates](https://kubernetes.io/blog/2022/12/26/pod-scheduling-readiness-alpha/) (alpha in Kubernetes 1.26). This controller acts as a translation layer between the Pod API and the Kueue Workload API. It implements queue admission by updating Pod tolerations.
+
+NOTE: Node taints should be pre-configured by cluster administrators.
+
+This directory should be considered a reference implementation of how Kueue can be extended out-of-tree to add support for different workload types.
+
+## Local Quickstart
+
+### Setup
+
+Create a kind cluster with Kueue installed.
+
+```bash
+./hack/kind-cluster.sh
+```
+
+Configure Kueue.
+
+```bash
+kubectl apply -f examples/queue.yaml
+kubectl apply -f examples/priorities.yaml
+```
+
+In another terminal, start the controller.
+
+```bash
+go run . --admission-taint-key=company.com/kueue-admission
```

Comment:
why not use the deployment instead?

Reply:
Rewrote guide to use a Deployment in-cluster